src/task/methods/ambros_m_judges_third_place/config.vsh.yaml

```
@@ -0,0 +1,56 @@
+__merge__: ../../api/comp_method.yaml
+
+functionality:
+  name: ambros_m_judges_third_place
+  info:
+    label: An ensemble model developed by AmbrosM
+    summary: "Ensemble comprised of four models: a py-boost model, an ExtraTrees model, and two recommender system models."
+    documentation_url: https://www.kaggle.com/competitions/open-problems-single-cell-perturbations/discussion/458661
+    repository_url: https://github.com/Ambros-M/Single-Cell-Perturbations-2023
+  arguments:
+#   - type: string
+#     name: --de_train
+#     default: "resources/neurips-2023-kaggle/de_train.parquet"
+#   - type: string
+#     name: --train_obs_zip
+#     default: "resources/neurips-2023-kaggle/train_obs.csv.zip"
+#   - type: string
+#     name: --id_map
+#     default: "resources/neurips-2023-kaggle/id_map.csv"
+#   - type: string
+#     name: --output
+#     default: "output.parquet"
+  resources:
+    - type: python_script
+      path: script.py
+platforms:
+  - type: docker
+    image: ghcr.io/openproblems-bio/base_python:1.0.4
```

This image already has cuda & pytorch installed

```suggestion
    image: ghcr.io/openproblems-bio/base_pytorch_nvidia:1.0.4
```